openbas-api/src/main/java/io/openbas/service/ScenarioService.java

```diff
@@ -296,6 +296,12 @@ public Scenario scenario(@NotBlank final String scenarioId) {
         .orElseThrow(() -> new ElementNotFoundException("Scenario not found"));
   }
 
+  public Scenario scenarioFromExerciseId(@NotBlank final String exerciseId) {
```

Comment:
scenarioFromExerciseId => scenarioFromSimulationId
String exerciseId => String simulationId
(I learned my lesson with chunk 2 avout this one ^^' )